src/mono/wasm/runtime/scheduling.ts

```diff
@@ -63,5 +69,8 @@ export function mono_wasm_schedule_timer(shortestDueTimeMs: number): void {
 }
 
 function mono_wasm_schedule_timer_tick() {
+    if (cwraps.mono_wasm_get_runtime_aborted())
```

Comment:
please add similar test into `fatal_handler` in `src\mono\wasm\runtime\loader\run.ts`

Reply:
Via runtimeHelpers same way as you did `set_runtime_aborted` also with checking that it was already set

Reply:
I'm just saying that `fatal_handler` doesn't have to try to exit if we are already aborted.